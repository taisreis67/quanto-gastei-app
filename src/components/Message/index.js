import React from 'react';

const Message = (props) => {
    const {
        textMessage,
        classMessage,
    } = props;

    return (
        <div className="row">
            <div className="col">
                <div className={`alert alert-${classMessage}`} role="alert">
                    {textMessage}
                </div>
            </div>
        </div>
    );
}

export default Message;