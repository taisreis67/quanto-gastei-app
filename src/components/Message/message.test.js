import React from 'react';
import Message from '../Message';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

it('Deve renderizar uma alerta com a menssagem passada.', () => {
    const message = mount(<Message textMessage="A message" classMessage="success" />);
    expect(toJson(message)).toMatchSnapshot();
});