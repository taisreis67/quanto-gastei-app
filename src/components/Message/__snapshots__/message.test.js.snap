// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar uma alerta com a menssagem passada. 1`] = `
<Message
  classMessage="success"
  textMessage="A message"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <div
        className="alert alert-success"
        role="alert"
      >
        A message
      </div>
    </div>
  </div>
</Message>
`;
